src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,72 @@
+package seedu.addressbook.ui;
+
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+
+import java.util.List;
+
+/**
+ * 
+ * Helper class to format strings appropriately.
+ */
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+    
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+    
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+    
+    /**
+     * Formats a string as a viewable indexed list item.
+     *
+     * @param visibleIndex visible index for this listing
+     */
+    public static String formatIndexedListItem(int visibleIndex, String listItem) {
+        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
+    }
+    
+    /** Formats a list of strings as a viewable indexed list. */
+    public static String formatIndexedListForViewing(List<String> listItems, int displayedIndexOffset) {
+        final StringBuilder formatted = new StringBuilder();
+        int displayIndex = 0 + displayedIndexOffset;
+        for (String listItem : listItems) {
+            formatted.append(formatIndexedListItem(displayIndex, listItem)).append("\n");
+            displayIndex++;
+        }
+        return formatted.toString();
+    }
+    
+    /** Formats string message to be displayed to user. 
+     * 
+     * @param string message displayed to user.
+     */
+    public static String formatMessage(String message) {
+        return LINE_PREFIX + message.replace("\n", LS + LINE_PREFIX);
+    }
+    
+    /** 
+     * Formats storage file message to be displayed to user. 
+     * 
+     * @param string storage file path
+     * */
+    public static String formatStorageFileInfo(String storageFilePath) {
+        return String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
+    }
+    
+    /**
+     * Formats command entered message to be displayed to user.
+     * 
+     * @param string full input line
+     */
+    public static String formatCommandEnteredMessage(String fullInputLine) {
+        return "[Command entered:" + fullInputLine + "]";
+    }
+    
+    /** Appends line prefix to string. */
+    public static String formatAddLinePrefix(String string) {
+        return LINE_PREFIX + string;
+    }
```

Comment:
Perhaps more formatting-related stuff (e.g. `DIVIDER`) can be moved to the `Formatter`? May be even things like this should be moved to the Formatter? Note that it may not always be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.